src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -130,6 +130,7 @@ export default {
 		KvLoadingOverlay,
 		LendHeader,
 		AddToBasketInterstitial,
+		lendFilterExpVersion: '',
```

Comment:
Please remove this line...I'm surprised you're not getting an error locally seeing as this isn't a component...

Reply:
Yeah, that's weird because I didn't see any compiling errors and also the linting checks not detected the bug

Reply:
Thanks for fixing this up!